old/lib/LedgerSMB/DBObject/User.pm

```diff
@@ -72,11 +71,9 @@ sub change_my_password {
     );
     if (!$verify){
         $self->error($self->{_locale}->text('Incorrect Password'));
+        die;
```

Comment:
Why this "die"? Doesn't $self->error do that already??

Reply:
Debugging leftover. Fixed